TEST: v1.16.x + blas variants

h-vetinari opened the pull request:
Following the same scheme as #295, #280, #273, #267, #242, #224, #223, #199, #172, #153 & #130.

(it's been 5 years of BLAS testing 😱)

h-vetinari committed:
test for all blas variants

h-vetinari committed:
enable setting pagefile size on windows

h-vetinari committed:
unskip tests that aren't due to excessive runtime

h-vetinari committed:
TST: don't build for osx-arm, no tests are run there

h-vetinari committed:
increase test suite verbosity due to hangs on windows

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.
 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.
 * ℹ️ scipy-tests output has some malformed specs:
- In section test: `libblas =*=*blas_impl`
Requirement spec fields should match the syntax `name [version [build]]`to avoid known issues in conda-build. For example, instead of `name =version=build`, use `name version.* build`. There should be no spaces between version operators and versions either: `python >= 3.8` should be `python >=3.8`.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/15984460448](https://github.com/conda-forge/conda-forge-webservices/actions/runs/15984460448). Examine the logs at this URL for more detail.</sub>

h-vetinari committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.21.08.15.33

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10
